### DISCUSSION TITLE
Correct OV2640 CAM type?

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#900] [Q&A] [answered]
Looking on https://de.aliexpress.com/item/1005003754592008.html resp. the image below, it seems that there are different OV2640 CAM types, eg. with different focus or night vision.
I haven't seen any information regarding this on this repo.

Could somebody share the experience/suggestions?

![grafik](https://user-images.githubusercontent.com/1783586/182955598-275dfe07-1585-4022-8075-f21ff74cd2af.png)

**Comment by microraptor** [comment, marked as answer]
Most ESP32-CAM boards seem to come with the 66deg lens. So that is probably what you want to get, if you want to be on the safe side.
